### DISCUSSION TITLE
How to implement a function similar to 'split output' in Imagica, as shown in the diagram above?

### DISCUSSION THREAD

**Discussion by zcc0511** [langgenius/dify#4179] [Help]
![2024 05 08_周传财](https://github.com/langgenius/dify/assets/93373579/c045f88e-3674-43f4-8e66-011b7696bee2)

How to implement a function similar to 'split output' in Imagica, as shown in the diagram above?

**Comment by xixici** [comment]
I think this can be resolved by splited prompts.
